.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "jenkins/dataproc-custom-images/dataproc-custom-images"]
```

Comment:
moved this from terra-docker, as we would like to build the dataproc image when the leonardo-terra-docker-build job updates the confs, and update the custom image reference in the confs.